README.md

```diff
@@ -10,11 +10,16 @@ can then perform whatever actions it wants.
 `snare` requires three command-line arguments to be specified:
 
 ```
-Usage: snare -p <port> -r <repo-programs-dir> -s <secrets-path>
+Usage: snare [-e email] [-j <maxjobs>] -p <port> -r <reposdir> -s <secretspath>
 ```
 
 where:
 
+ * `<email>` is an (optional) email address to which any errors running per-repo
+   programs will be sent (warning: full stderr/stdout will be sent, so consider
```

Comment:
Had to double take this sentence, but I can't see a rewording which makes it any easier.